include/openssl/bnerr.h

```diff
@@ -72,7 +72,6 @@ int ERR_load_BN_strings(void);
 # define BN_F_BN_SET_WORDS                                144
 # define BN_F_BN_STACK_PUSH                               148
 # define BN_F_BN_USUB                                     115
-# define BN_F_OSSL_BN_RSA_DO_UNBLIND                      151
```

Comment:
This change looks wrong in a public header

Reply:
Hmm... right... on the other hand this was there only for one patch release so I cannot imagine anybody being broken by it disappearing.

Reply:
> Hmm... right... on the other hand this was there only for one patch release so I cannot imagine anybody being broken by it disappearing.

I'm not so sure, but you might be right. I personally think we should just reinstate these macros, because the cost of having them is low. But if we choose to keep them deleted then an OTC exception should be sought.

Reply:
Yeah, adding it back seems to be the easiest way. I'll do a PR.

Reply:
https://github.com/openssl/openssl/pull/20784